operate/client/src/App/DecisionInstance/index.test.tsx

```diff
@@ -261,6 +261,7 @@ describe('<DecisionInstance />', () => {
     mockFetchDrdData().withSuccess(mockDrdData);
     mockFetchDecisionDefinitionXML().withSuccess(mockDmnXml);
     mockFetchDecisionInstance().withSuccess(invoiceClassification);
+    mockFetchDecisionInstanceV2().withSuccess(invoiceClassificationV2);
```

Comment:
Can we remove the old mock?

Reply:
It's remove in the next PR (https://github.com/camunda/camunda/pull/38762). Right now it still has to be loaded because the decision instance store and thus the old API is still required by the DRD data store. 

https://github.com/camunda/camunda/pull/38870 deletes the old mock function completely.